applications/StructuralMechanicsApplication/structural_mechanics_application.h

```diff
@@ -420,11 +426,15 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) KratosStructuralMechanicsAppl
     // Line load
     const LineLoadCondition<2> mLineLoadCondition2D2N;
     const LineLoadCondition<2> mLineLoadCondition2D3N;
+    const LineLoadCondition<2> mLineLoadCondition2D4N;
+    const LineLoadCondition<2> mLineLoadCondition2D5N;
     const LineLoadCondition<3> mLineLoadCondition3D2N;
     const LineLoadCondition<3> mLineLoadCondition3D3N;
 
     const SmallDisplacementLineLoadCondition<2> mSmallDisplacementLineLoadCondition2D2N;
     const SmallDisplacementLineLoadCondition<2> mSmallDisplacementLineLoadCondition2D3N;
+    const SmallDisplacementLineLoadCondition<2> mSmallDisplacementLineLoadCondition2D4N;
```

Comment:
It would be interesting adding some patch tests for the corresponding elements. See the tests in the structural mechanics as example

Reply:
in the form of cpp as unit-test? Or real separate test cases?

Reply:
I would extend the current patch_tests in the tests folder

Reply:
Added patch tests